crates/bevy_animation/src/graph.rs

```diff
@@ -172,6 +183,96 @@ pub enum AnimationGraphLoadError {
     SpannedRon(#[from] SpannedError),
 }
 
+/// Acceleration structures for animation graphs that allows Bevy to evaluate
+/// them quickly.
+///
+/// These are kept up to date as [`AnimationGraph`] instances are added,
+/// modified, and removed.
+#[derive(Default, Reflect, Resource)]
+pub struct ThreadedAnimationGraphs(
+    pub(crate) HashMap<AssetId<AnimationGraph>, ThreadedAnimationGraph>,
+);
+
+/// An acceleration structure for an animation graph that allows Bevy to
+/// evaluate it quickly.
+///
+/// This is kept up to date as the associated [`AnimationGraph`] instance is
+/// added, modified, or removed.
+#[derive(Default, Reflect)]
+pub(crate) struct ThreadedAnimationGraph {
+    /// A cached postorder traversal of the graph.
+    ///
+    /// The node indices here are stored in postorder. Siblings are stored in
```

Comment:
I know you document this below, but I think it would be useful to say upfront, right at the top of this doc comment, that it's stored in post-order *with children being iterated in reverse* (`4, 3, 6, 5, 2, 1` instead of `5, 6, 2, 3, 4, 1`)

Reply:
It does say "siblings are stored in descending order".